mmdeploy/backend/base/backend_manager.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import importlib
+from abc import ABCMeta, abstractstaticmethod
+from typing import Any, Optional, Sequence
+
+
+class BaseBackendManager(metaclass=ABCMeta):
+    """Abstract interface of backend utils."""
+
+    @abstractstaticmethod
```

Comment:
It looks like being deprecated since python3.3, referring to https://docs.python.org/3/library/abc.html